source/socketstuff.lua

```diff
@@ -11,183 +11,192 @@ gbolIsAHost = false                					-- defaults to NOT a host until the play
 	end
 	return retvalue
 end
 
-function socketstuff.GetItemInClientQueue()
+
+function socketstuff.getItemInClientQueue()
 -- returns the first/oldest item in the message queue and deletes that item from the queue
 
 	local retval
-	if #arrClientIncomingQueue > 0 then
-		retvalue = arrClientIncomingQueue[1]
-		table.remove(arrClientIncomingQueue,1)
+	if #clientIncomingQueue > 0 then
+		retvalue = clientIncomingQueue[1]
+		table.remove(clientIncomingQueue,1)
 	else
 		return nil
 	end
 	return retvalue
 end
 
-function socketstuff.AddItemToClientOutgoingQueue(message)
+
+function socketstuff.addItemToClientOutgoingQueue(message)
 -- adds item to the array for later sending
 
 	if message ~= nil then
-		table.insert(arrClientOutgoingQueue, message)
+		table.insert(clientOutgoingQueue, message)
 	end
 end
 
-function socketstuff.AddItemToHostOutgoingQueue(message)
+
+function socketstuff.addItemToHostOutgoingQueue(message)
 -- adds item to the array for later sending
 
 	if message ~= nil then
-		table.insert(arrHostOutgoingQueue, message)
+		table.insert(hostOutgoingQueue, message)
 	end
 end
 
-function socketstuff.SendToHost()
+
+function socketstuff.sendToHost()
 -- send the whole outgoing queue to the host
 
-	while #arrClientOutgoingQueue > 0 do
-		if arrClientOutgoingQueue[1] ~= nil then
-			local serialdata = bitser.dumps(arrClientOutgoingQueue[1])
-			udpclient:send(serialdata)
+	while #clientOutgoingQueue > 0 do
+		if clientOutgoingQueue[1] ~= nil then
+			local serialdata = bitser.dumps(clientOutgoingQueue[1])
+			udpClient:send(serialdata)
 		end
-		table.remove(arrClientOutgoingQueue,1)
+		table.remove(clientOutgoingQueue,1)
 	end
 end
 
-function socketstuff.SendToClients()
--- sends the whole outgoing queue to all of the clients
-	while #arrHostOutgoingQueue > 0 do
-		if arrHostOutgoingQueue[1] ~= nil then
 
-			local serialdata = bitser.dumps(arrHostOutgoingQueue[1])
-			for k,v in pairs(arrClientNodes) do
-				udphost:sendto(serialdata, v.ip, v.port)
-print(#arrClientNodes)
+function socketstuff.sendToClients()
+-- sends the whole outgoing queue to all of the clients
+	while #hostOutgoingQueue > 0 do
+		if hostOutgoingQueue[1] ~= nil then
+		
+			local serialdata = bitser.dumps(hostOutgoingQueue[1])
+			for k,v in pairs(cilentNodes) do
+				udpHost:sendto(serialdata, v.ip, v.port)
 			end
 		end
-		table.remove(arrHostOutgoingQueue,1)
+		table.remove(hostOutgoingQueue,1)
 	end
 end
 
-function socketstuff.ConnectToHost(IPAddress, IPPort)
+
+function socketstuff.connectToHost(IPAddress, IPPort)
 -- Client has decided to connect to host
---! IPAddress is not used and probably should be
+-- TODO: implement IPAddress
 
     -- set up a client connect
     local address, port = "localhost", IPPort
 
-    udpclient = socket.udp()
-    udpclient:settimeout(0)
-    udpclient:setpeername(address, port)
+    udpClient = socket.udp()
+    udpClient:settimeout(0)
+    udpClient:setpeername(address, port)
     gbolIsAClient = true

```

Comment:
Not PR related:
`gbolIsAClient ` and `gbolIsAHost` might could be one variable in the future, like `isHosting = true`, since if the client isn't hosting, it has to be a client. I thought about something like `myInstance.isHosting` or `client.isHosting` but I think all of them are quite bad ... why is this so difficult lol.

Reply:
Can be a host or a client or none (single player)

Reply:
none also a client in my book :grin: . But you'll know what's best here for sure.